src/olympia/stats/search.py

```diff
@@ -142,7 +144,7 @@ def reindex_tasks_group(index_name):
     from olympia.stats.management.commands.index_stats import (
         gather_index_stats_tasks
     )
-    return gather_index_stats_tasks(index_name)
+    return group(gather_index_stats_tasks(index_name))
```

Comment:
Goes with the change mentioned above, for consistency both `reindex_tasks_group` functions for stats and addons return a flat `group` of tasks.